tools/mktutorial/main.go

```diff
@@ -186,9 +186,9 @@ func gatherTutorials(root string) ([]tutorial, error) {
 
 		// Great! We have a new tutorial. Append it and let's move on to the next one.
 		tutorials = append(tutorials, tutorial{
-			Name:              name,
-			Title:             title,
-			H1:                h1,
+			Name:  name,
```

Comment:
Nit: whitespace change -- something up with your IDE perhaps?

Reply:
This is actually correct `gofmt` formatting - not sure why it wasn't formatted originally.